extension/js/common/api/google.ts

```diff
@@ -554,6 +554,7 @@ export class GoogleAuth {
       profile: 'https://www.googleapis.com/auth/userinfo.profile', // needed so that `name` is present in `id_token`, which is required for key-server auth when in use
       compose: 'https://www.googleapis.com/auth/gmail.compose',
       modify: 'https://www.googleapis.com/auth/gmail.modify',
+      contactsReadOnly: 'https://www.googleapis.com/auth/contacts.readonly',
       contacts: 'https://www.google.com/m8/feeds/',
```

Comment:
If what you do ends up working as expected, the original `contacts` scope will be outdated, so you can remove it.

Reply:
Okay, will do, thanks!